### DISCUSSION TITLE
求助：npm i/yarn完成之后npm start不了

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#192] [问题求助] [answered]
> # npm start
> cq-picsearcher-bot@2.29.0 start /root/cq-picsearcher-bot
> pm2 start ecosystem.config.js

[PM2] Applying action restartProcessId on app [cqps](ids: [ 0 ])
[PM2][ERROR] Process 0 not found
Process 0 not found
/root/cq-picsearcher-bot/node_modules/pm2/lib/API.js:1720
        acted.push(proc.pm2_env ? proc.pm2_env.pm_id : proc.pm_id)
                        ^

TypeError: Cannot read property 'pm2_env' of undefined
    at /root/cq-picsearcher-bot/node_modules/pm2/lib/API.js:1720:25
    at Array.forEach (<anonymous>)
    at API.speedList (/root/cq-picsearcher-bot/node_modules/pm2/lib/API.js:1719:18)
    at /root/cq-picsearcher-bot/node_modules/pm2/lib/binaries/CLI.js:301:15
    at wrapper (/root/cq-picsearcher-bot/node_modules/async/internal/once.js:12:16)
    at replenish (/root/cq-picsearcher-bot/node_modules/async/internal/eachOfLimit.js:76:25)
    at iterateeCallback (/root/cq-picsearcher-bot/node_modules/async/internal/eachOfLimit.js:65:17)
    at /root/cq-picsearcher-bot/node_modules/async/internal/onlyOnce.js:12:16
    at /root/cq-picsearcher-bot/node_modules/pm2/lib/binaries/CLI.js:292:11
    at /root/cq-picsearcher-bot/node_modules/pm2/lib/API.js:334:21
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cq-picsearcher-bot@2.29.0 start: `pm2 start ecosystem.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cq-picsearcher-bot@2.29.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-05-23T03_05_13_207Z-debug.log

**Comment by Tsuk1ko** [comment, marked as answer]
npx pm2 kill 后再试
